samples/apps/logging/logging.cpp

```diff
@@ -1,11 +1,15 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the Apache 2.0 License.
-#include "enclave/app_interface.h"
+
+// This app's includes
 #include "formatters.h"
 #include "logging_schema.h"
-#include "node/historical_queries_adapter.h"
-#include "node/quote.h"
-#include "node/rpc/metrics_tracker.h"
+
+// CCF
+#include "apps/utils/metrics_tracker.h"
+#include "ccf/app_interface.h"
+#include "ccf/historical_queries_adapter.h"
+#include "http/http_query.h"
 #include "node/rpc/user_frontend.h"
```

Comment:
I think the core of "what are we trying to do" here, is to reach a point where all of the CCF includes here (and in other apps) are of public `ccf/` files. We can make an exception for some shared utils that several of our apps use, like `metrics_tracker.h`. The next step after that, to stop installing our private headers, is to add enough forward declaration/hidden implementation that the public headers also only include public headers, though that's not so urgent.